### PR TITLE
DF-199 config updated to get rid of the legacy oracle and replace it …

### PR DESCRIPTION
**Pull request by dvoracek-martin**
…with postgresql

### CODE REVIEW

File: application-prod.yml [hunk 1]
```diff
@@ -104,16 +104,6 @@ infrastructure:
   database:
     postgres:
       hostname: 10.129.15.122
-    oracle-legacy:
```
Comment (fabiangotzen) [inline application-prod.yml]: Please keep in mind that we have a deployment planned for next week. So if we merge this into the master branch we should be sure that everything works without the oracle db.

File: addressservice.yml [hunk 2]
```diff
@@ -4,9 +4,9 @@ spring:
     platform: ORACLE
     driver-class-name: oracle.jdbc.OracleDriver
```
Comment (birom) [inline addressservice.yml]: The ``basisbatches`` db is PostgreSQL not Oracle. The properties ``platform`` and ``driver-class-name`` also must be changed.
Do we have the needed table and data in the ``basisbatches`` DB that is used here? ``oracle-egov`` is not the same as the ``oracle-legacy``.

File: addressservice.yml [hunk 3]
```diff
@@ -4,9 +4,9 @@ spring:
...
     type: com.zaxxer.hikari.HikariDataSource
-    url: ${infrastructure.database.oracle-egov.url}
-    username: ${infrastructure.database.oracle-egov.username}
-    password: ${infrastructure.database.oracle-egov.password}
+    url: ${infrastructure.database.basisbatches.url}
+    username: ${infrastructure.database.basisbatches.username}
+    password: ${infrastructure.database.basisbatches.password}
   jpa:
     database-platform: org.hibernate.dialect.Oracle10gDialect
```
Comment (birom) [inline addressservice.yml]: Oracle dialect is used here instead of a PostgreSQL.

File: reference-jobcenter-importer.yml [hunk 4]
```diff
@@ -12,9 +12,9 @@ reference-jobcenter-importer:
   datasource:
     jobcenter:
       type: com.zaxxer.hikari.HikariDataSource
-      url: ${infrastructure.database.oracle-legacy.url}
-      username: ${infrastructure.database.oracle-legacy.username}
-      password: ${infrastructure.database.oracle-legacy.password}
+      url: ${infrastructure.database.basisbatches.url}
```
Comment (birom) [inline reference-jobcenter-importer.yml]: Please do not forget to remove the Oracle JDBC driver dependency from the project.

File: candidate-importer.yml [hunk 5]
```diff
@@ -12,9 +12,9 @@ candidate-importer:
     datasource:
         stes:
             type: com.zaxxer.hikari.HikariDataSource
```
Comment (birom) [inline candidate-importer.yml]: Please do not forget to remove the Oracle JDBC driver dependency from the project.

**Comment by seco-mgabor** [comment]
Marton, do you think we still need this branch / pull request?

**Comment by birom** [comment]
I close this pr. If we migrate the ``getaux`` app and we have the ``addresses``, I will open a new actual one.

**Pull request closed without merging**